py-polars/tests/unit/io/test_ipc.py

```diff
@@ -234,7 +234,7 @@ def test_from_float16() -> None:
 def test_binview_ipc_mmap(tmp_path: Path) -> None:
     df = pl.DataFrame({"foo": ["aa" * 10, "bb", None, "small", "big" * 20]})
     file_path = tmp_path / "dump.ipc"
-    df.write_ipc(file_path, future=True)
+    df.write_ipc(file_path)
```

Comment:
I want to ensure we use the latest version here. If the `write_ipc` would change, this test would not be testing anymore what it should test.

Reply:
`future=True` was the default for `write_ipc` before this PR. Highest flavor is still used as the default for `write_ipc`. I removed it because it is not necessary.

Reply:
I know, but if we change ICP to not use `newest`, this test doesn't test what it should test anymore. Can you make the `newest` explicit?